internal/k8sdeps/transformer/hash/namehash_test.go

```diff
@@ -20,7 +20,7 @@ import (
 	"reflect"
 	"testing"
 
-	"sigs.k8s.io/kustomize/internal/k8sdeps"
+	"sigs.k8s.io/kustomize/internal/k8sdeps/unstructadapter"
```

Comment:
Can you make this just 'unstruct' (or kunstruct, or kunstructured). 

No need to call out the adapter aspect of one of the objects in the package.

Actually, could you send one PR that does the splitting of k8sdeps into multiple packages, and just that?

Reply:
That sounds good. I will split k8sdeps in a different PR.